Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
+COPY . /oopt-gnpy
+WORKDIR /oopt-gnpy
+RUN python setup.py install
+WORKDIR /shared/examples
+ENTRYPOINT ["/oopt-gnpy/.docker-entry.sh"]
+CMD "/bin/bash"
```

Comment:
Minor detail perhaps but is there a reason you're not using the preferred _exec_ form of CMD, i.e. `CMD ["/bin/bash"]`?

Reply:
Thanks, #297